### ISSUE TITLE
More translation enchancements - util.inc, user.inc, team.inc

### ISSUE THREAD

**Issue by romw** [BOINC/boinc-ng#984]
**Reported by Simek on 17 Apr 40683750 21:20 UTC**
More translation enhancements for: "util.inc", "user.inc" and "team.inc".

**Comment by romw** [comment]
**Commented by Simek on 22 Aug 40684269 09:20 UTC**
Additional patch which fixes two existing translation strings.

**Comment by romw** [comment]
**Commented by ChristianB on 2 Oct 40686489 00:26 UTC**
Please be aware when enabling translations for send_email() or other stuff that sends a notification via email to the user that the language is determined by the '''senders''' browser preferences. If the sender selected Spanish and the receiver German because he can't speak Spanish the message head or information will be in Spanish! This is also applicable for #1006

**Comment by romw** [comment]
**Commented by davea on 28 May 40686620 22:13 UTC**
Thanks.  For future reference, please:
1) don't change coding conventions (e.g. curly brackets)
2) use "" for all strings; use \" as needed
3) test all changes for correctness if possible

-- David

**Comment by romw** [comment]
**Commented by davea on 17 Aug 40686622 02:40 UTC**
(In [22319]) - web: make strings translatable.  From Simek, fixes #1007

**Comment by romw** [comment]
**Commented by Simek on 15 Feb 40686670 15:33 UTC**
In my honest opinion the code need a lot of clean up, but what is the future? What with Drupal module? Should I concentrate on cleaning and improving existing BOINC Web platform code or should I help some who works on module?

I really would like to test all changes but I've got a lot of problems when I tried to create test project for this so I gave up.

It will be nice if I found someone who give me an access to FTP "html" folder on his test project.
